tests/baselines/reference/argumentExpressionContextualTyping.errors.txt

```diff
@@ -27,8 +25,7 @@ tests/cases/conformance/expressions/contextualTyping/argumentExpressionContextua
     
     baz(array);                          // Error
         ~~~~~
-!!! error TS2345: Argument of type '(string | number | boolean)[]' is not assignable to parameter of type '[string, number, boolean]'.
-!!! error TS2345:   Property '0' is missing in type '(string | number | boolean)[]'.
+!!! error TS2735: Type '(string | number | boolean)[]' is missing the following properties from type '[string, number, boolean]': 0, 1, 2

```

Comment:
Maybe separately, we should do better on tuples since this is so stupid.

Reply:
It's better than what it is now; but yeah, having a dedicated "array is not assignable to tuple" message might be an decent idea.